### PR TITLE
Update data viz palette

### PR DESCRIPTION
**Pull request by krkshitij**
## Previous Behavior

Some of the colors in data viz palette don't match with the colors in design

## New Behavior

Colors in data viz palette match with the colors in design

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/64d15aac652f3562a2a45cee638cb56efcd2b600'>64d15aac652f3562a2a45cee638cb56efcd2b600</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/utilities/colors.ts [hunk 1]
```diff
@@ -90,18 +90,18 @@ const QualitativePalette: Palette = {
...
   '38': ['#23330b', '#a4cc6c'], // [lime.shade40, lime.tint30],
   '39': ['#63276d', '#cf87da'], // [lilac.shade30, lilac.tint30],
-  '40': ['#3a2f00', '#ecdfa5'], // [gold.shade40, gold.tint40],
+  '40': ['#3a2f00', '#dac157'], // [gold.shade40, gold.tint30],
 };
 
 const SemanticPalette: Palette = {
   info: ['#015cda'],
-  disabled: ['#8a8886'],
+  disabled: ['#dbdbdb', '#4d4d4d'], // [grey[86], grey[30]]
```
Comment (AtishayMsft) [inline packages/react-charting/src/utilities/colors.ts:104]: disabled light is #bdbdbd

File: packages/react-charting/docs/colors.md [hunk 2]
```diff
@@ -66,7 +66,10 @@ The users can pick colors from our data visualization color palette, which is ca
 
 ### Qualitative Color Palette
 
-![Qualitative Color Palette](images/colors/1.png)
+![First 10 colors in Qualitative palette](images/colors/1.png)
+![Second 10 colors in Qualitative palette](images/colors/2.png)
```
Comment (AtishayMsft) [inline packages/react-charting/docs/colors.md:70]: slot 20 is named as slot 10. I have asked Karina to fix it.

Comment (krkshitij) [reply packages/react-charting/docs/colors.md:70]: okay

Comment (AtishayMsft) [reply packages/react-charting/docs/colors.md:70]: This is fixed. Can you refresh the image

File: packages/react-charting/src/utilities/colors.ts (hunk 1 above)
Comment (krkshitij) [reply packages/react-charting/src/utilities/colors.ts:104]: Inspect and the tokens package says it should be #dbdbdb

image

image

File: packages/react-charting/docs/colors.md (hunk 2 above)
Comment (krkshitij) [reply packages/react-charting/docs/colors.md:70]: done

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**